### PR TITLE
(stoa0040)

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Combined stoa0040.stoa003 and stoa0040.stoa004-one work previously split into two TEI-XML files
Edited some earlier Augustine metadata to current standard.

### CODE REVIEW

File: data/stoa0040/stoa074/__cts__.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <ti:title xml:lang="lat">Quaestiones in Heptateuchum</ti:title>
   <ti:edition  xml:lang="lat" workUrn="urn:cts:latinLit:stoa0040.stoa074" urn="urn:cts:latinLit:stoa0040.stoa074.opp-lat1">
     <ti:label xml:lang="lat">Quaestiones in Heptateuchum</ti:label>
-    <ti:description xml:lang="mul">Augustine.  Sancti Aureli Augustini Opera, Sectio III, Pars 3. 
-      Zycha, Joseph, editor. Corpus scriptorum ecclesiasticorum Latinorum, Volume 28.2, 1896.</ti:description>
+    <ti:description xml:lang="mul">Augustine.  Sancti Aureli Augustini Opera, Sectio III, Pars 3. (Corpus scriptorum ecclesiasticorum Latinorum, Volume 28.2.)
+      Zycha, Joseph, editor. Prague; Vienna; Leipzig: F. Tempsky; G. Freytag, 1895..</ti:description>
```
Comment (lcerrato) [inline data/stoa0040/stoa074/__cts__.xml:6]: duplicate period here

**Review by lcerrato — CHANGES_REQUESTED** [review]
@AlisonBabeu 
The files are too large for me to review within GitHub interface.

**Comment by lcerrato** [comment]
@AlisonBabeu 
Give me a chance to look over the large file offline.

**Comment by lcerrato** [comment]
@AlisonBabeu 
The combination looks good!
I don't know why they have line breaks in here, but that seems to be a convention of the data entry? There's a lot of other conventions that are odd, but I think for CSEL this works.

File: data/stoa0040/stoa074/__cts__.xml (hunk 1 above)
Comment (AlisonBabeu) [reply data/stoa0040/stoa074/__cts__.xml:6]: I fixed this so I'm going to merge this pull request since the other file looks ok.

**Pull request merged**